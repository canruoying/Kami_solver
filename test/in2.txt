1222111331
1132333311
1132111331
4444441133
4132141441
4132244411
4133211441
4113222244
2213333333
1221111213
1122211223
1221211123
2211213321
4113333121
4411211121
1441222221