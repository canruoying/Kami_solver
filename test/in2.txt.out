=======
1 2 2 2 1 1 1 3 3 1 
1 1 3 2 3 3 3 3 1 1 
1 1 3 2 1 1 1 3 3 1 
4 4 4 4 4 4 1 1 3 3 
4 1 3 2 1 4 1 4 4 1 
4 1 3 2 2 4 4 4 1 1 
4 1 3 3 2 1 1 4 4 1 
4 1 1 3 2 2 2 2 4 4 
2 2 1 3 3 3 3 3 3 3 
1 2 2 1 1 1 1 2 1 3 
1 1 2 2 2 1 1 2 2 3 
1 2 2 1 2 1 1 1 2 3 
2 2 1 1 2 1 3 3 2 1 
4 1 1 3 3 3 3 1 2 1 
4 4 1 1 2 1 1 1 2 1 
1 4 4 1 2 2 2 2 2 1 
=======
 ||
 ||
 || Step 1
\||/
 \/
=======
1 2 2 2 1 1 1 3 3 1 
1 1 3 2 3 3 3 3 1 1 
1 1 3 2 1 1 1 3 3 1 
4 4 4 4 4 4 1 1 3 3 
4 1 3 2 1 4 1 4 4 1 
4 1 3 2 2 4 4 4 1 1 
4 1 3 3 2 1 1 4 4 1 
4 1 1 3 2 2 2 2 4 4 
4 4 1 3 3 3 3 3 3 3 
1 4 4 1 1 1 1 2 1 3 
1 1 4 4 4 1 1 2 2 3 
1 4 4 1 4 1 1 1 2 3 
4 4 1 1 4 1 3 3 2 1 
4 1 1 3 3 3 3 1 2 1 
4 4 1 1 2 1 1 1 2 1 
1 4 4 1 2 2 2 2 2 1 
=======
 ||
 ||
 || Step 2
\||/
 \/
=======
1 2 2 2 1 1 1 3 3 1 
1 1 3 2 3 3 3 3 1 1 
1 1 3 2 1 1 1 3 3 1 
3 3 3 3 3 3 1 1 3 3 
3 1 3 2 1 3 1 3 3 1 
3 1 3 2 2 3 3 3 1 1 
3 1 3 3 2 1 1 3 3 1 
3 1 1 3 2 2 2 2 3 3 
3 3 1 3 3 3 3 3 3 3 
1 3 3 1 1 1 1 2 1 3 
1 1 3 3 3 1 1 2 2 3 
1 3 3 1 3 1 1 1 2 3 
3 3 1 1 3 1 3 3 2 1 
3 1 1 3 3 3 3 1 2 1 
3 3 1 1 2 1 1 1 2 1 
1 3 3 1 2 2 2 2 2 1 
=======
 ||
 ||
 || Step 3
\||/
 \/
=======
1 2 2 2 1 1 1 1 1 1 
1 1 1 2 1 1 1 1 1 1 
1 1 1 2 1 1 1 1 1 1 
1 1 1 1 1 1 1 1 1 1 
1 1 1 2 1 1 1 1 1 1 
1 1 1 2 2 1 1 1 1 1 
1 1 1 1 2 1 1 1 1 1 
1 1 1 1 2 2 2 2 1 1 
1 1 1 1 1 1 1 1 1 1 
1 1 1 1 1 1 1 2 1 1 
1 1 1 1 1 1 1 2 2 1 
1 1 1 1 1 1 1 1 2 1 
1 1 1 1 1 1 1 1 2 1 
1 1 1 1 1 1 1 1 2 1 
1 1 1 1 2 1 1 1 2 1 
1 1 1 1 2 2 2 2 2 1 
=======
 ||
 ||
 || Step 4
\||/
 \/
=======
2 2 2 2 2 2 2 2 2 2 
2 2 2 2 2 2 2 2 2 2 
2 2 2 2 2 2 2 2 2 2 
2 2 2 2 2 2 2 2 2 2 
2 2 2 2 2 2 2 2 2 2 
2 2 2 2 2 2 2 2 2 2 
2 2 2 2 2 2 2 2 2 2 
2 2 2 2 2 2 2 2 2 2 
2 2 2 2 2 2 2 2 2 2 
2 2 2 2 2 2 2 2 2 2 
2 2 2 2 2 2 2 2 2 2 
2 2 2 2 2 2 2 2 2 2 
2 2 2 2 2 2 2 2 2 2 
2 2 2 2 2 2 2 2 2 2 
2 2 2 2 2 2 2 2 2 2 
2 2 2 2 2 2 2 2 2 2 
=======
Time Elapsed: 53273ms
